server/src/main/java/io/crate/protocols/postgres/PostgresNetty.java

```diff
@@ -135,6 +144,7 @@ public PostgresNetty(Settings settings,
         this.nettyBootstrap = nettyBootstrap;
         this.transport = netty4Transport;
         this.pageCacheRecycler = pageCacheRecycler;
+        this.activeSessions = new PgSessions(new ConcurrentHashMap<>(), transportKillJobsNodeAction);
```

Comment:
Any reason to pass in the sessions map? I think `PgSessions` should keep it internal to not leak implementation details